pkg/cluster/operation/action.go

```diff
@@ -441,6 +475,52 @@ func RestartComponent(getter ExecutorGetter, instances []spec.Instance, timeout
 	return nil
 }
 
+func enableInstance(getter ExecutorGetter, ins spec.Instance, timeout int64, isEnable bool) error {
+	e := getter.Get(ins.GetHost())
+	if isEnable {
+		log.Infof("\tEnabling instance %s %s:%d", ins.ComponentName(), ins.GetHost(), ins.GetPort())
+	} else {
+		log.Infof("\tDisabling instance %s %s:%d", ins.ComponentName(), ins.GetHost(), ins.GetPort())
+	}
+
+	action := "disable"
+	if isEnable {
+		action = "enable"
+	}
+
+	// Enable/Disable by systemd.
+	c := module.SystemdModuleConfig{
+		Unit:    ins.ServiceName(),
+		Action:  action,
+		Timeout: time.Second * time.Duration(timeout),
+	}
+	systemd := module.NewSystemdModule(c)
+	stdout, stderr, err := systemd.Execute(e)
+
+	if len(stdout) > 0 {
+		fmt.Println(string(stdout))
```

Comment:
Will this broke the output on parallel executing?

Reply:
I think no. Because the `start`, `restart`, `stop` action works well with this.